### PR TITLE
Shubham/charter methods

### PR DESCRIPTION
**Pull request by ShubhamSukum**

### CODE REVIEW

File: src/charts/chartsModel.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    data: {
+      type: [Number],
+      required: true,
+    },
+  },
+  {
+    versionKey: false,
+    timestamps: true,
+  }
+);
```
Comment (SiddheshDhinge) [inline src/charts/chartsModel.js:18]: Use this model for all charts

File: src/charts/chartsModel.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        type: [Number],
+        required: true,
+      },
+    },
+  },
+  {
+    versionKey: false,
+    timestamps: true,
+  }
+);
```
Comment (SiddheshDhinge) [inline src/charts/chartsModel.js:81]: remove this schemas

File: src/charts/chartsModel.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+export const barModel = mongoose.model("bars", commonSchema);
+export const pieModel = mongoose.model("pies", commonSchema);
+export const histModel = mongoose.model("hists", histSchema);
```
Comment (SiddheshDhinge) [inline src/charts/chartsModel.js:85]: give complete names

File: src/charts/chartsModel.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+export const histforNo7Model = mongoose.model("sevens", histforNo7Schema);
+export const q9Model = mongoose.model("nines", q9Schema);
```
Comment (SiddheshDhinge) [inline src/charts/chartsModel.js:87]: remove

File: src/charts/Q5Model.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      },
+    },
+  },
+  {
+    versionKey: false,
+    timestamps: true,
+  }
+);
+
+export const Q5model = mongoose.model("fives", Q5schema);
```
Comment (SiddheshDhinge) [inline src/charts/Q5Model.js:95]: Name it 'CompositePieChart'

File: src/matches/matches.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+          .then((savedDocument) => {
+            return console.log("Document saved successfully:", savedDocument);
+          })
+          .catch((error) => {
+            return console.error("Error saving document:", error);
+          });
+      } catch (e) {
+        console.error(e);
+      }
+}
```
Comment (SiddheshDhinge) [inline src/matches/matches.js:16]: Comment there that this is only for testing purposes

File: src/matches/matchesModel.js [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "pies",
+        required: true,
+      },
+      Q4_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "bars",
+        required: true,
+      },
+      No_5: {
```
Comment (SiddheshDhinge) [inline src/matches/matchesModel.js:31]: fix typo

**Review by SiddheshDhinge — COMMENTED** [review]
Please look at this changes
otherwise LGTM

File: src/charts/chartsModel.js (hunk 4 above)
Comment (ShubhamSukum) [reply src/charts/chartsModel.js:87]: I had to keep the  "nines" collection as we changed our plan

File: src/charts/Q5Model.js (hunk 5 above)
Comment (ShubhamSukum) [reply src/charts/Q5Model.js:95]: named it as "compositepiecharts" as only this sits in mongoDB rules

**Pull request closed without merging**